01week/rockPaperScissors.js

```diff
@@ -1,47 +1,92 @@
-function playFunction() {
-  var userChoice = prompt("Do you choose rock, paper or scissors?");
-  var computerChoice = Math.random();
-  if (computerChoice < 0.34) {
-      computerChoice = "rock";
-  } else if(computerChoice <= 0.67) {
-      computerChoice = "paper";
-  } else {
-      computerChoice = "scissors";
-  }
-  
-  var compare = function(choice1, choice2) {
-      if(choice1 === choice2) {
-        alert("The result is a tie!");
-  }
-  if(choice1 === "rock") {
-      if(choice2 === "scissors") {
-          alert("rock wins");
-      } else {
-          alert("paper wins");
-      }
-  }
-  if(choice1 === "paper") {
-      if(choice2 === "rock") {
-          alert("paper wins");
-      } else {
-          if(choice2 === "scissors") {
-              alert("scissors wins");
-      }
-  }
-  if(choice1 === "scissors") {
-      if(choice2 === "rock") {
-          alert("rock wins");
-      } else {
-          if(choice2 === "paper") {
-             alert("scissors wins");
+'use strict';
+
+const assert = require('assert');
+const readline = require('readline');
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+
+function rockPaperScissors(hand1, hand2) {
+    
+    console.log(hand1,hand2,)
+    if(hand1 === hand2) {
+            return ("It's a tie!");
           }
+    else if(hand1==='rock' && hand2 ==='scissors'){
+          return('Hand one wins!')
+        }
+    else if (hand1==='paper' && hand2 === 'rock'){
+        return('Hand one wins!')
+        }
+        else if(hand1==='scissors' && hand2 === 'paper'){
+            return('Hand one wins!')
+        }
+
+    if(hand2 === hand1) {
+        return ("The result is a tie!");
       }
-  }
-  }
-  };
-  console.log("User Choice: " + userChoice);
-  console.log("Computer Choice: " + computerChoice);
-  compare(userChoice, computerChoice)
-  } 
-
-  
\ No newline at end of file
+else if(hand2==='rock' && hand1 ==='scissors'){
+      return('Hand two wins!')
+    }
+else if (hand2==='paper' && hand1 === 'rock'){
+    return('Hand two wins!')
+    }
+    else if(hand2==='scissors' &&hand1 === 'paper'){
+        return('Hand two wins!')
+    }
```

Comment:
Looks great so far. But I think you aren't completely checking this test "should scrub input to ensure lowercase with "trim"ed whitepace'"